lib/src/repository/git_local_repository.dart

```diff
@@ -64,6 +78,11 @@ class GitLocalRepository {
       environment: {
         'LANG': 'C', // default English locale that is always present
         'LC_ALL': 'en_US',
+        // prevent git from reading host configuration files
+        'HOME': homePath,
```

Comment:
Lets set `GIT_TERMINAL_PROMPT: '0'`

See: https://git-scm.com/docs/git

That's stop it from prompting for anything.